src/internal/m365/collection/drive/handler_utils.go

```diff
@@ -99,9 +88,9 @@ func augmentItemInfo(
 			Modified:   ptr.Val(item.GetLastModifiedDateTime()),
 			Owner:      creatorEmail,
 			ParentPath: pps,
-			SiteID:     siteID,
+			SiteID:     resource.ID(),
```

Comment:
Ash was bringing up a probability of this causing issues in Ark. I'm not sure how this gets consumed in Ark, but would be useful to check.

Reply:
Good point, I'll call it in with them.

Reply:
I had a chat. Looks like this is not a problem.